### ISSUE TITLE
Dynamically Request Log Level?

### ISSUE THREAD

**Issue by da3dsoul** [toddmeinershagen/NLog.SignalR#14]
This is a question and feature request, I guess. Is there a way to have a client request the minimum Log Level and have the hub respond dynamically only when that Log Level is hit?

I'm impatient and just started with the project, so I may figure it out myself. If I do, I'll post it here for posterity. Maybe it could added to the wiki or sample. IDK

**Comment by manofthetent** [comment]
@da3dsoul I wanted you to know that the SignalRTarget does not set up the Hub.  This target establishes a HubProxy for connecting to the SignalR Hub at the Uri and Name that is specified for the target.  The library also provides the common ILoggingHub interface for the SignalR service set up.

I include a diagram to show how it currently works.

![image](https://user-images.githubusercontent.com/20459187/89718918-24da6800-d991-11ea-863c-bce945042c42.png)

So - the target is not handling the Hub behavior.  If we wanted to have the Client communicate a log level to the Hub, we would need to enhance the ILoggingHub interface and provide an implementation of the LoggingHub to provide that behavior.

**Comment by da3dsoul** [comment]
Yeah I figured it out after the fact. What I meant was dynamically via query strings, but I'm not sure that's ideal after my experience with SignalR.
